client-registry-api/src/main/java/uk/gov/di/authentication/clientregistry/entity/ClientRegistrationResponse.java

```diff
@@ -48,7 +51,8 @@ public ClientRegistrationResponse(
             @JsonProperty(value = "post_logout_redirect_uris") List<String> postLogoutRedirectUris,
             @JsonProperty(value = "back_channel_logout_uri") String backChannelLogoutUri,
             @JsonProperty(required = true, value = "service_type") String serviceType,
-            @JsonProperty(required = true, value = "subject_type") String subjectType) {
+            @JsonProperty(required = true, value = "subject_type") String subjectType,
+            @JsonProperty(required = true, value = "claims") List<String> claims) {
```

Comment:
does this need to be mandatory ? what about non id services

Reply:
Fixed